plugins/resources/plugin.utils.js

```diff
@@ -94,7 +94,7 @@ function toRelativePath(pathToFile, pathToParent){
 function getTempLocations(config){
   const contractsRoot = path.parse(config.contractsDir).dir
   const cwd = config.workingDir;
-  const contractsDirName = '.coverage_contracts';
+  const contractsDirName = config.coverage_contracts_temp || '.coverage_contracts';
```

Comment:
```suggestion
  const contractsDirName = config.coverageContractsTemp || '.coverage_contracts';
```

Could we make this camel-cased, like the other config options? (Will need to be reflected in the docs as well...)

Reply:
Sure, will update and add to the docs